docs/Configuration.md

```diff
@@ -90,7 +54,7 @@ Here is an example key file:
 Use the JSON file to generate a Kubernetes secret.
 
 ```shell
-kubectl create secret generic fe-creds --from-file=credentials=./<JSON file name>
+kubectl -n upbound-system create secret generic fe-creds --from-file=credentials=./<JSON file name>
```

Comment:
In all examples we are use `crossplane-system`

Reply:
Yes but the marketplace is in the Upbound side. Crossplane is the open source project.

Reply:
it is therefore necessary to plan to replace all namespace in examples

Reply:
I'm created issue to reference this : #90